### PR TITLE
Update Feast API endpoints

### PR DESCRIPTION
**Pull request by ariefrahmansyah**
**What this PR does / why we need it**:

Calls to Feast endpoints are not working anymore since Merlin using outdated endpoints.

https://user-images.githubusercontent.com/8122852/196861307-44056e5f-efcf-42f2-a4bb-7201dbc136be.mov

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Checklist**

- [ ] Added unit test, integration, and/or e2e tests
- [x] Tested locally
- [ ] Updated documentation
- [ ] Update Swagger spec if the PR introduce API changes
- [ ] Regenerated Golang and Python client if the PR introduce API changes

### CODE REVIEW

File: ui/src/providers/feast/FeastResourcesContext.js [hunk 1]
```diff
@@ -5,28 +5,25 @@ const FeastResourcesContext = React.createContext({});
...
-    feastEndpoints.listFeatureTables,
-    { method: "POST", muteError: true },
+    `${feastEndpoints.listFeatureTables}?project=${project}`,
+    { method: "GET", muteError: true },
     {},
     false
   );
 
   useEffect(() => {
+    console.log("project:", project);
```
Comment (khorshuheng) [inline ui/src/providers/feast/FeastResourcesContext.js:22]: This is probably added by mistake

Comment (ariefrahmansyah) [reply ui/src/providers/feast/FeastResourcesContext.js]: Thanks for pointing this. I've removed it

**Comment by khorshuheng** [comment]
Sorry, i miss this during reviewing: the response return is likely different from what the frontend javascript expects. Can we go through the parts that relies on the response, and see if the response is parsed correctly?

**Comment by ariefrahmansyah** [comment]
> Sorry, i miss this during reviewing: the response return is likely different from what the frontend javascript expects. Can we go through the parts that relies on the response, and see if the response is parsed correctly?

I've tested it on dev. The response structure is still same and it's working correctly.

**Comment by khorshuheng** [comment]
> 

Ok, in that case this PR lgtm.

**Review by pradithya — APPROVED** [review]
LGTM, thanks @ariefrahmansyah !

**Pull request merged**